docs/Analyzer Configuration.md

```diff
@@ -298,3 +298,12 @@ Option Values: Integer values of System.Runtime.InteropServices.DllImportSearchP
 Default Value: Specific to each configurable rule ('770', which is AssemblyDirectory | UseDllDirectoryForDependencies | ApplicationDirectory, by default for most rules)
 
 Example: `dotnet_code_quality.CA5392.unsafe_DllImportSearchPath_bits = 770`
+
+#### Configure if only look at Controller-derived classes when considering CSRF
```

Comment:
Is this an option specific to Dataflow analysis? I think we should move this option, and even `unsafe_DllImportSearchPath_bits` and `sufficient_IterationCount_for_weak_KDF_algorithm` above the Dataflow analysis section: https://github.com/dotnet/roslyn-analyzers/blob/master/docs/Analyzer%20Configuration.md#dataflow-analysis

Reply:
Can you also add "Configurable Rules" entry like https://github.com/dotnet/roslyn-analyzers/pull/2857/files#diff-1b5cdc4937a8cdfc79545945ad6c0783R251 so we are explicit about which CA rule specifies which option? It would be good to specify it even for prior to options.

Reply:
Instead of just "Controller", I think we should have the fully qualified class name so it's not ambiguous with the older ASP.NET.

Also, tagging @gewarren for wording suggestions